### DISCUSSION TITLE
Force Add A Username?

### DISCUSSION THREAD

**Discussion by CynicalPlatapus** [MalloyDelacroix/DownloaderForReddit#366] [Ideas]
Since pretty much every username i'm adding at the moment comes back with "Invalid Name", can we have a feature to force add the name to the list anyway, so i can come back to it later and try to perform a download when it might be working?

It'd be far easier than what i'm currently doing, i.e. keeping a notepad file of usernames to try re-adding another time.

**Comment by MalloyDelacroix** [comment]
This has been added in version 3.16.0.  There's now a checkbox on the add user/subreddit dialog that controls validation.

**Reply by CynicalPlatapus** [reply]
much appreciated
